subprojects/jacoco/src/main/java/org/gradle/testing/jacoco/tasks/JacocoReportBase.java

```diff
@@ -59,17 +56,7 @@
     private final ConfigurableFileCollection additionalSourceDirs = getProject().files();
 
     public JacocoReportBase() {
-        onlyIf(new Spec<Task>() {
-            @Override
-            public boolean isSatisfiedBy(Task element) {
-                return Iterables.any(getExecutionData(), new Predicate<File>() {
-                    @Override
-                    public boolean apply(File file) {
-                        return file.exists();
-                    }
-                });
-            }
-        });
```

Comment:
To replicate this behavior, I think we have to mark the execution data with `@SkipWhenEmpty`

Reply:
This was an accidental removal; I reverted